plume-admin-api-log/src/main/java/com/coreoz/plume/admin/LogEntryTransformer.java

```diff
@@ -13,12 +13,10 @@
 
 /**
  * Represent an operation that transforms the trace generated by {@link OkHttpLoggerInterceptor}
- * <p>
- * This is a functional interface</a>
+ * <p>This is a <a href="package-summary.html">functional interface</a>
  * whose functional method is {@link #transform(Request, Response, LogInterceptApiBean)}.
- * <p>
- * The static method {@link #limitBodySizeTransformer(int)} transforms the body of either the request or the response
- * to be truncated at a given limit
+ *
+ * <p>The static method {@link #emptyTransformer()} initiates the transformer
```

Comment:
Peut être que tu pourrais le formuler autrement : 

> There are multiple ways to create a transformer:
- Starting from the empty transfomer: {@code LogEntryTransformer.emptyTransformer()}
- Starting using an existing transformer: {@code LogEntryTransformer.limitBodySizeTransformer(1000)} or {@code LogEntryTransformer.hideJsonFields(...)} or {@code LogEntryTransformer.emptyBody()}